.eslintrc.json

```diff
@@ -8,7 +8,8 @@
   "rules": {
     "indent": [
       "error",
-      2
+      2,
+      { "MemberExpression": 0 }
```

Comment:
Please do not modify project's `eslintrc`. It'll be updated when we'll update to eslint v4 (it's not needed now)